receiver/hostmetricsreceiver/internal/scraper/loadscraper/load_scraper_test.go

```diff
@@ -51,7 +56,7 @@ func TestScrape(t *testing.T) {
 
 	for _, test := range testCases {
 		t.Run(test.name, func(t *testing.T) {
-			scraper := newLoadScraper(context.Background(), zap.NewNop(), &Config{})
+			scraper := newLoadScraper(context.Background(), zap.NewNop(), &Config{PerCPU: test.perCPU})
```

Comment:
You need a test as well to assert that the resulting Load1/5/15 will be accurate.

Reply:
the assert is already done below no?

Reply:
Kind of: the fact that your test didn't catch the bug about using the multiplier operator instead of divider proves that the test wasn't sufficient.

Reply:
hello @jpkrohling, I added a test to catch this kind of error. probably not ideal but I do not have better idea to check this. please give me your thoughts